File: app/src/main/java/com/typ/muslim/features/prays/PrayNotifyMethodChangedCallback.java
package com.typ.muslim.features.prays;

import com.typ.muslim.features.prays.enums.Prays;
import com.typ.muslim.enums.PrayNotifyMethod;

public interface PrayNotifyMethodChangedCallback {

	void onPrayNotifyMethodChanged(Prays forPray, PrayNotifyMethod newMethod);

}

File: app/src/main/java/com/typ/muslim/features/prays/PrayNotifyMethodChangedCallback.kt
package com.typ.muslim.features.prays

import com.typ.muslim.enums.PrayNotifyMethod
import com.typ.muslim.features.prays.enums.PrayType

interface PrayNotifyMethodChangedCallback {
    fun onPrayNotifyMethodChanged(forPray: PrayType, newMethod: PrayNotifyMethod)
}
